src/main/resources/mustache-templates/layout/footer.html

```diff
@@ -1,19 +1,37 @@
+
+<footer class="page-footer font-small mdb-color lighten-3 pt-4">
 	<div class="container">
-		<footer>
-			<p class="pull-right"><small>HPO on FHIR</small></p>
-		</footer>
+		<div class="row">
+			<div class="col-sm-6 footer-copyright py-3">
+				© 2018: <a href="https://github.com/OCTRI/poc-hpo-on-fhir"> HPO
+					on FHIR</a>
+			</div>
+			<div class="col-sm-6">
+				<div class="float-right">
+					<a href="http://www.ohsu.edu/xd/research/centers-institutes/octri/">
+						<img src="/images/octri.gif" height="30" class="d-inline-block"
+						alt="">
```

Comment:
Might be nice to have a little something for the `alt` text. Maybe just OCTRI logo. Similarly for the `jax.png`.